wallet-new/integration/Functions.hs

```diff
@@ -517,8 +522,8 @@ respToRes resp = do
 
 
 -- | Pick a random element using @IO@.
-pickRandomElement :: (MonadIO m, HasCallStack) => [a] -> m a
-pickRandomElement [] = error "called with empty list"
+pickRandomElement :: (MonadIO m, Alternative m) => [a] -> m a
+pickRandomElement [] = empty
```

Comment:
We were getting some `elements: called with empty list` errors. This will turn those showstoppers into test skips.

Reply:
with this we can remove all explicit `guard`s